### PR TITLE
Bug 1236248 - Fix new_activity intermittent tests

### PR DESCRIPTION
**Pull request by azasypkin**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=95808d45f2bb03074d39fb34ed9239bdadcf556d

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1ebc74ad1a015f81b74819df56e27fdfd4531cdc

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=74cb5c6b8eb0e4170432d2c07dd070f5f7459b96

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d41041345fb3f05cff5950daee0ea40b95112463

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c57f6a6c35e7cc381d40dba76b94e9696c0c109d

File: apps/sms/views/shared/js/utils.js [hunk 1]
```diff
@@ -940,7 +940,12 @@
         });
 
       return promise.then(
-        () => shimHostIframe.contentWindow.bootstrap(appInstanceId)
+        () => {
+          window.mozSetMessageHandler = function(type, handler) {
```
Comment (azasypkin) [inline apps/sms/views/shared/js/utils.js]: Just first idea that came to my mind, maybe we can think of something smarter - just want to finally enable our tests :)

File: tests/taskcluster/tasks.yml [hunk 2]
```diff
@@ -14,29 +14,5 @@ params:
...
-    chunks: 40
-
-  marionette_js_tv_tests:
-    chunks: 10
-
-  unit_tests_b2g:
-    chunks: 30
-
-  jsmarionette_unit_tests:
+    chunks: 1
```
Comment (julienw) [inline tests/taskcluster/tasks.yml]: you'll take care to revert this ;)

File: tests/taskcluster/tasks/marionette_js_tests.yml [hunk 3]
```diff
@@ -19,6 +19,7 @@ task:
       REPORTER: 'mocha-tbpl-reporter'
       TEST_MANIFEST: '{{gaia}}/shared/test/integration/tbpl-manifest.json'
       NODE_MODULES_SRC: npm-cache
+      APP: sms
```
Comment (julienw) [inline tests/taskcluster/tasks/marionette_js_tests.yml]: same

File: apps/sms/views/shared/js/utils.js (hunk 1 above)
Comment (julienw) [inline apps/sms/views/shared/js/utils.js]: that works for me, but please add a reference to the bug # + some explanation in a comment

Comment (azasypkin) [inline apps/sms/views/shared/js/utils.js]: Yep, sure!

File: tests/taskcluster/tasks.yml (hunk 2 above)
Comment (azasypkin) [inline tests/taskcluster/tasks.yml]: Ha-ha, yeah :)

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=84100fc54b611182c93700bd9cb06eafc8fd0808

**Pull request merged**